code/game/sound.dm

```diff
@@ -165,4 +165,10 @@
 				soundin = pick('sound/items/bikehorn.ogg', 'sound/items/AirHorn2.ogg', 'sound/misc/sadtrombone.ogg', 'sound/items/AirHorn.ogg', 'sound/effects/reee.ogg',  'sound/items/WEEOO1.ogg', 'sound/voice/beepsky/iamthelaw.ogg', 'sound/voice/beepsky/creep.ogg','sound/magic/Fireball.ogg' ,'sound/effects/pray.ogg', 'sound/voice/hiss1.ogg','sound/machines/buzz-sigh.ogg', 'sound/machines/ping.ogg', 'sound/weapons/flashbang.ogg', 'sound/weapons/bladeslice.ogg')
 			if("goose")
 				soundin = pick('sound/creatures/goose1.ogg', 'sound/creatures/goose2.ogg', 'sound/creatures/goose3.ogg', 'sound/creatures/goose4.ogg')
+			if("warpspeed")
+				soundin = 'sound/effects/hyperspace_begin.ogg'
```

Comment:
I don't see where you use `"warpspeed"` or where you add the ogg file.

Reply:
me ne

> I don't see where you use `"warpspeed"` or where you add the ogg file.

me neither.

Reply:
That... that means you should remove these two lines...

Reply:
> 
> 
> That... that means you should remove these two lines...

done